_spinnaker_install_admin_guides/generating-certificates.md

```diff
@@ -13,54 +13,59 @@ This guide includes helpful commands to get started with self-signed certificate
 
 ## Prerequisites
 
-You will need a recent version of OpenSSL.
+You need a recent version of OpenSSL.
 
 
-## Generate self-signed certificate authority
+## Generating self-signed certificate authority
 
-Let's generate a key for our certificate authority. Replace `TRUSTSTORE_PASS` with your own CA password.
+Generate a key for our certificate authority:
 
-```bash
```

Comment:
I removed bash from the codeblocks cause the current CSS renders some of it really poorly (pink txt and a dark background: https://share.getcloudapp.com/6quBRQPK).